<?xml version="1.0"?>
<launch>
  <arg name="output" default="screen"/>
  <arg name="respawn" default="false"/>
  <arg name="start_manager" default="true"/>
  <arg name="start_server" default="false"/>
  <arg name="nodelet_manager" default="mummer_nodelet_manager"/>
  <arg name="launch-prefix" default=""/>

  <arg name="default_inputs" default="robot/merged"/>
  <arg name="output_world" default="robot/monitored"/>

  <arg name="alpha" default="0.1"/>
  <arg name="area_threshold" default="0.70"/>

  <arg name="y_min" default="7.658"/>
  <arg name="y_max" default="11.657"/>
  <arg name="x_min" default="25.22"/>
  <arg name="x_max" default="30.31"/>

  <node pkg="uwds_mummer_clients" type="area_monitor.py" name="area_monitor" output="screen">
    <rosparam subst_value="true">
      default_inputs: $(arg default_inputs)
      output_world: $(arg output_world)
      x_min: $(arg x_min)
      x_max: $(arg x_max)
      y_min: $(arg y_min)
      y_max: $(arg y_max)
      alpha: $(arg alpha)
      area_threshold: $(arg area_threshold)
    </rosparam>
  </node>

</launch>
